contributor-guide/modules/ROOT/pages/version-control.adoc

```diff
@@ -7,51 +7,125 @@ Boost uses the Git version control system for its development. This allows multi
+The official Boost repositories are hosted in GitHub under the https://github.com/boostorg/[] account. Each library has its own repository under this organization. For example, https://github.com/boostorg/json for boost:json[], or https://github.com/boostorg/date_time/ for boost:date_time[].
 
-As with any Git-based project, you can clone the repositories, make changes, commit those changes to your local repository, and push your changes back to the server. If you want to contribute changes to the official Boost libraries, you can make a pull request, which the maintainers of the libraries can then review and potentially merge into the official codebase. Be aware though that Boost has a well-defined workflow for contributors, including guidelines for how to submit changes, how changes are reviewed, and so forth. Refer to xref:getting-involved.adoc[].
+As with any Git-based project, you can clone the repositories, make changes, commit those changes to your local repository, and push your changes back to the server. If you want to contribute changes to the official Boost libraries, refer to xref:getting-involved.adoc#contribute[Contribute to an Existing Library].
 
 == The Super-project Repository
 
 The Boost https://github.com/boostorg/boost[*Super-project*] repository includes all the libraries as submodules.
 
-In addition to the libraries, you'll find many meta-repositories under `boostorg` that handle other tasks, for example:
+In addition to the libraries, you'll find many other repositories under `boostorg` that handle other tasks, for example:
 
 * https://github.com/boostorg/website2022[Website generation]
-* https://github.com/boostorg/cmake[Cmake support]
+* https://github.com/boostorg/cmake[CMake support]
 * https://github.com/boostorg/regression[Regression testing]
 * https://github.com/boostorg/release-tools[Release tools]
 
-There is also a https://github.com/boostorg/boost/tree/master/tools[tools] directory in the Super-project, which contains various tools used for tasks such as building and testing Boost libraries. 
+CMake is one of the most popular of the https://github.com/boostorg/boost/tree/master/tools[boost/tools] directory. You will find other tools used for tasks such as building and testing Boost libraries. 
 
-The *master* branch of the Super-project is usually the development version of Boost, where new features are added and bugs are fixed. When there is a public release of Boost, it is built from the Super-project repo.
+The Super-project has both *master* and *develop* branches, which have the same semantics as these branches do for individual libraries. New features are added and bugs are fixed in the *develop* branch. When the *develop* branch is complete, it is merged with the *master* branch. When there is a public release of Boost, it is built from the *master* branch after that branch has been closed to updates (refer to the xref:release-process.adoc[]).
```

Comment:
> When the *develop* branch is complete, it is merged with the *master* branch.

This is unfortunately technically incorrect.

- For individual libraries, when the *develop* branch is complete, it is merged with the *master* branch.
- For the super-project, the develop and master branches are independent. They track the latest changes in master/develop in all libraries. There's no merging involved because this strategy makes merging impossible.

I argued against this strategy myself but it's how it is.

Reply:
Ah, OK, added an explanation